packages/renderer/src/lib/EngineIcon.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+import Fa from 'svelte-fa/src/fa.svelte';
+import { faCircle } from '@fortawesome/free-solid-svg-icons';
+export let engine = '';
+let color = '';
+
+if (engine === 'Podman') {
+  color = 'text-purple-400';
+} else if (engine === 'Docker') {
+  color = 'text-blue-400';
```

Comment:
The colors should not be hardcoded
it needs to be provided by the extension.

Reply:
So in the extension should we hardcore that the extension is "blue" and that we get it from there?

Reply:
probably and provide two colors, one for light and one for dark.
this is why probably changing the way on how the badge for the engine is rendered should be a separate PR